plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleStateInterceptor.java

```diff
@@ -33,7 +34,7 @@ public HttpServerHandleStateInterceptor(TraceContext traceContext, MethodDescrip
     }
 
     public boolean validate(Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```

Comment:
I think `args.length < 2` would be better to change `ArrayUtils.getLength(args) < 2`, args could be 0 size arrays.